src/lib/daemon_rpcs/types.ml

```diff
@@ -175,7 +176,7 @@ module Status = struct
       | Check_again of Block_time.Stable.Latest.t
       | Produce of producing_time
       | Produce_now of producing_time
-      | Evaluating_vrf of Mina_numbers.Global_slot.Stable.Latest.t
+      | Evaluating_vrf of Mina_numbers.Global_slot_since_genesis.Stable.Latest.t
```

Comment:
This shouid be `Global_slot_since_hard_fork.t` but since this is used only for display, might make sense leave it as is or add `Global_slot_since_hard_fork.t` in addition to `Global_slot_since_genesis.t`

Reply:
I changed it to `Global_slot_since_hard_fork.t`